content/en/docs/High Availability/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "High Availability"
+linkTitle: "High Availability"
+weight: 170
+description: >
+  Learn how to configure High Availability in Kyverno
+---
+
+## Configure Kyverno in HA mode
+To install Kyverno in HA using Helm Chart.
+  
+  **NOTE:** Due to some cmplexities accompanied with running 2 replicas, the recommended replica counts for HA is at least 3.
+
+``` 
+helm install kyverno kyverno/kyverno --create-namespace --set=replicaCount=3 
+```
+## How HA works in Kyverno
+This section provides details on how Kyverno handles HA scenario.
+- ### Module - Webhook Server
+  Webhook server is where Kyverno receives and processes admission requests. This controller does not require a leader-election. When Kyverno runs multiple instances, the Service will distribute the admission requests across different instances. 
+
+  For “mutate” and “validate” enforce policies, Kyverno returns the decision along with the admission response; it is a synchronous request-response process. However, for “generate” and “validate” audit policies, Kyverno pushes these requests to a queue, and returns the response immediately, and then starts processing the data asynchronously. 
+
+  If the process is terminated, we need to complete pending requests / drain the queue and then shutdown Kyverno gracefully.
+
+  **Updated:** the queue in the validate audit handler is used to generate policy reports. Since the report will be reconciled when Kyverno restarts, there’s no need to drain this queue on shutdown.
```

Comment:
Why the Updated note here? Updated for what? This can probably be eliminated and folding in the text after L24.